Directory.Build.targets

```diff
@@ -23,6 +24,8 @@
 
   <!-- IsGrpcPublishedPackage is set in csproj so related config must be in targets file -->
   <ItemGroup Condition="'$(IsGrpcPublishedPackage)' == 'true'">
+    <None Include="$(PackageIconFullPath)" Pack="true" PackagePath="\"/>
```

Comment:
What is this for?

Reply:
The docs recommend this: https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-an-icon-image-file